library/alloc/src/collections/btree/set.rs

```diff
@@ -18,8 +15,6 @@ use super::Recover;
 
 use crate::alloc::{Allocator, Global};
 
-// FIXME(conventions): implement bounded iterators
-
```

Comment:
I think this refers `range`, which was added. [Originated](https://github.com/rust-lang/rust/pull/18605/files#diff-7bd3ef40bf3a7606987b34184852640b709b62013c731cb411aca5011b405facR23) in #18605. Similar comment for map was removed since.